src/constants/flawFields.ts

```diff
@@ -44,4 +47,10 @@ export const flawFields = fieldsFor(ZodFlawSchema)
 
 export const allowedEmptyFieldMapping: Record<string, string> = {
   'cve_id': 'cve_id__isempty',
+  'cvss_scores__score': 'cvss_scores__score__isempty',
```

Comment:
As discussed in https://github.com/RedHatProductSecurity/osidb/pull/579, the filter for CVSS is specific for every issuer/version pair. So we have `cvss2_rh__isempty`, `cvss3_nist__isempty`, etc.

Reply:
Okay